### PR TITLE
lockup module accumulation store invariant

### PR DESCRIPTION
**Pull request by antstalepresh**
Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

File: x/lockup/keeper/invariants.go [hunk 1]
```diff
@@ -34,3 +36,40 @@ func SyntheticLockupInvariant(keeper Keeper) sdk.Invariant {
...
 	}
 }
+
+func AccumulationStoreInvariant(keeper Keeper) sdk.Invariant {
+	return func(ctx sdk.Context) (string, bool) {
+		moduleAcc := keeper.ak.GetModuleAccount(ctx, types.ModuleName)
+		balances := keeper.bk.GetAllBalances(ctx, moduleAcc.GetAddress())
+
+		// check 1s, 1 day, 1 week, 2 weeks
+		durations := []time.Duration{time.Second, time.Hour * 24, time.Hour * 24 * 7, time.Hour * 24 * 14}
```
Comment (alexanderbez) [inline x/lockup/keeper/invariants.go:46]:
```suggestion
		durations := []time.Duration{
		    time.Second,
		    time.Hour * 24,
		    time.Hour * 24 * 7,
		    time.Hour * 24 * 14,
		 }
```

File: x/lockup/keeper/invariants.go [hunk 2]
```diff
@@ -34,3 +36,40 @@ func SyntheticLockupInvariant(keeper Keeper) sdk.Invariant {
 		return sdk.FormatInvariant(types.ModuleName, "synthetic-lockup-invariant", "All synthetic lockup invariant passed"), false
 	}
 }
+
+func AccumulationStoreInvariant(keeper Keeper) sdk.Invariant {
```
Comment (alexanderbez) [inline x/lockup/keeper/invariants.go:40]: nit: let's add a godoc for this pls

**Review by alexanderbez — APPROVED** [review]
LGTM

**Review by p0mvn — APPROVED** [review]

**Comment by p0mvn** [comment]
Hi @antstalepresh . What is the status of this?

**Comment by p0mvn** [comment]
Seeing this has been open for a while.

If this is still in progress, let's mark it as a draft. If not planning to merge, let's close, please

File: x/lockup/keeper/invariants.go [hunk 3]
```diff
@@ -34,3 +36,45 @@ func SyntheticLockupInvariant(keeper Keeper) sdk.Invariant {
...
```
Comment (ValarDragon) [inline x/lockup/keeper/invariants.go:40]:
```suggestion
// AccumulationStoreInvariant ensures that the sum of all lockups at a given duration
// is equal to the value stored within the accumulation store.
func AccumulationStoreInvariant(keeper Keeper) sdk.Invariant {
```

**Review by ValarDragon — APPROVED** [review]

**Review by alexanderbez — APPROVED** [review]

**Review by alexanderbez — APPROVED** [review]

**Comment by czarcas7ic** [comment]
Yes I just fixed it just need to merge @alexanderbez

**Pull request merged**